docker-compose.prod.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.3"
+services:
+  flight_review_app:
+    # image: px4flightreview
+    build:
+      context: ./app
+      dockerfile: Dockerfile
+    restart: always
+    env_file: .env
+    volumes:
+      - ./data:/opt/service/data
+      - ./app/config_default.ini:/opt/service/config_default.ini:ro
+      - ./app/config_user.ini:/opt/service/config_user.ini:ro
+  certbot:
+    image: certbot/certbot
+    volumes:
+      - ./data/certbot/conf:/etc/letsencrypt
+      - ./data/certbot/www:/var/www/certbot
+    entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew; sleep 12h & wait $${!}; done;'"
```

Comment:
Can this use crond?

Reply:
Yes I would like to test this certbot in a aws server, i will change to crond once I tested it work.